internal/db/diff/diff.go

```diff
@@ -82,6 +82,7 @@ func DiffSchema(ctx context.Context, source, target string, schema []string, p u
 			utils.DifferImage,
 			nil,
 			args,
+			nil,
```

Comment:
these changes are intentional?

Reply:
yup, due to introducing a new `binds` arg to the function signature.